### ISSUE TITLE
OTU taxa picking

### ISSUE THREAD

**Issue by philipbaileynar** [namc-utah/namc-bug-db#154]
Trip, the first taxa in PIBO is *Acari*. When I look in `PilotDB.BugOTU` the topmost taxa that is matched to Acari is the Class *Arachnida*. The subclass *Acari* is also matched of course, but I was hoping to use SQL to find the highest level taxa that matches the items in your spreadsheet.

**Question** what is the rollup taxa for Acari in the PIBO OTU please?**

Here's the top 5 taxa from `PilotDB.BugOTU` that match *Acari* in the OTUNamePIBO column:

| OTUCodeNone | OTUNameNone | OTUCodePIBO | OTUNamePIBO | taxalevel |
| :--- | :--- | :--- | :--- | :--- |
| 56 | Arachnida | 7 | Acari | Class |
| 2789 | Acari | 7 | Acari | SubClass |
| 55 | Sarcoptiformes | 7 | Acari | Order |
| 58 | Trombidiformes | 7 | Acari | Order |
| 1821 | Araneae | 7 | Acari | Order |

And here's a snapshot of your spreadsheet:

<img width="466" alt="Screen Shot 2021-06-09 at 4 24 33 PM" src="https://user-images.githubusercontent.com/8645379/121441722-2ffa7d80-c93f-11eb-8283-4b745b006ff6.png">

**Comment by Feldspar5** [comment]
If I understand correctly, Acari is listed in three different taxa levels( class, sub-class, and order).  This clearly reveals an issue with our taxonomy.  We do not have original documents (apart from Alaska, maybe one other) that indicate the original designers' mapping if there is ambiguity.  I suggest that we use Andrew's QCed taxonomy table and rely on that moving forward.  If a name does not appear on his list (odd grouping etc.) we would need to use our best judgement for rolling based on the BugOTU table from Pilot.  This table theoretically represents a mapping as originally intended.  Apparently not clearly though.  Perhaps you could run through all OTUs for all models and provide a list of ambiguous taxa connections for me and Andrew to review.  Hopefully there are not too many.  

Additionally, I propose we change 'FinalOTU' to 'refOTU' in the new database.

**Comment by Feldspar5** [comment]
I agree that this needs to be QCed by NAMC, but I propose NAR prepopulates for Andrew and pass to him for review.  I suspect that the majority of refOTUs will map correctly.  After all, it is the stored procedure from that BugOTU table that does the mapping for us.  We have no other _better_ source of information.

**Comment by philipbaileynar** [comment]
@Feldspar5  what is the column in the `BugOTU` table that corresponds with the OTU called `OTUCodeOR_NorthernBasinRange`?

**Comment by philipbaileynar** [comment]
@Feldspar5  in the ORNBR worksheet you have a row with `Epeorus, Zaitzevia`. Is this a cut and paste error and meant to be two taxa? Or should the comma simply be a space?

**Comment by philipbaileynar** [comment]
@Feldspar5 you have just one Colorado worksheet called "CO MMI" but we have three Colorado OTUs in the new database that correspond nicely to three columns in BugOTU:

|New Database OTU Name|BugOTU Column|
|---|---|
|OTUCodeCO09|OTUnameCO9|
|OTUCodeCOedas|OTUnameCOedas|
|OTUCodeCOedas17|OTUCodeCOedas17|

The BugOTU records look different for the three old OTUs.

Is there meant to be just one OTU for all three Colorado models or does each model have a separate OTU?

**Comment by Feldspar5** [comment]
Epiorus and zaitzevia are separate taxa

**Comment by Feldspar5** [comment]
The coedas (no 17) should be the correct ones

**Comment by Feldspar5** [comment]
The or nbr model is a null model with only the few taxa listed

**Comment by philipbaileynar** [comment]
Thanks Trip. 

The software is ready for Andrew to review the OTU taxa. Closing this issue.
